### PR TITLE
Jest

### PR DESCRIPTION
**Pull request by DimitarHristovski**

### CODE REVIEW

File: components/test/component.test.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+test('Jest Test Example', () => {
+  expect(true).toBe(true);
+});
```
Comment (edichoska) [inline components/test/component.test.js:3]: we are using Typescript in the project. Please use the .ts instead of .js

Comment (stoimenov95) [reply components/test/component.test.js]: @DimitarHristovski can you please provide another example related to the app itself, e.g. a test for some of the existing components (Render the component with some text prop, assert that the expected text is in the document)

**Review by stoimenov95 — CHANGES_REQUESTED** [review]

File: components/test/component.test.js (hunk 1 above)
Comment (DimitarHristovski) [reply components/test/component.test.js]: Theres no code to be tested yet or if there is give me an example and im gonna change it

Comment (stoimenov95) [reply components/test/component.test.js]: @DimitarHristovski you can add a test for the page component in the app root folder and check if the h1 is rendering with the text 'Home LearnHub.mk'. Everything else regarding the setup seems all right to me, @edichoska what do you say?

**Pull request closed without merging**